blazor/gantt-chart/tooltip.md

```diff
@@ -525,4 +525,168 @@ A manual taskbar tooltip can be customized using the [GanttTooltipSettings.Manua
 }
 ```
 
-![Blazor Gantt Chart displays Manual Taskbar Tooltip](images/blazor-gantt-chart-manual-taskbar-tooltip.png)
\ No newline at end of file
+![Blazor Gantt Chart displays Manual Taskbar Tooltip](images/blazor-gantt-chart-manual-taskbar-tooltip.png)
+
+### Indicator tooltip
```

Comment:
1. Add mouse cursor in images.
2. Move these two topics to summar.md file under main toolip